### PR TITLE
Don't react to PR comments

### PR DESCRIPTION
**Pull request by foolip**
This simplifies the filtering and removes the need to fetch the pull
request in `getPullRequest`.

### CODE REVIEW

**Comment by foolip** [comment]
The code changes here are probably straightforward enough to review, but the key concern here should be unintended side effects of this change:
- Could the code be broken in such a way that comments were previously the *only* thing that triggered the right things, given that a PR description counts as a comment?
- Is wpt-pr-bot reliable, and if not is that effectively masked by reacting to comments also?

**Comment by jugglinmike** [comment]
I like this simplification! We should also reconfigure the GitHub Webhook by removing the "Issues" and "Issue comments" events. In that case, I'd push to remove the explicit filtering (since that would be dead code, and it would insinuate the possibility of unreachable states).

> - Could the code be broken in such a way that comments were previously the *only* thing that triggered the right things, given that a PR description counts as a comment?
> - Is wpt-pr-bot reliable, and if not is that effectively masked by reacting to comments also?

I'm not aware of any problems like that, but I can't say they don't exist. We could build some confidence by inspecting ["timelines"](https://developer.github.com/v3/issues/timeline/) from recent pull requests, looking for places where the script acted directly after a comment.

**Comment by foolip** [comment]
Yes, that's a good point. Perhaps that could then be isolated from the code that reacts to PR creation/update, though? It was quite hard to follow the code as it was here.

**Comment by zcorpan** [comment]
> Perhaps that could then be isolated from the code that reacts to PR creation/update, though?

It probably could, yeah.

**Comment by foolip** [comment]
I still see https://github.com/web-platform-tests/wpt-pr-bot/issues/77 as plan A, certainly for adding labels and reviewers. @zcorpan's upcoming work on improving the review process might reveal some needs that we can't do with GitHub Actions and maybe we'll have to keep operating this bot, but most of the code that exists I'm not confident will continue to live in this repo. That is to say, improving as we make changes is a good idea, but wrangling it all under control could well turn out to be a wasted effort.

**Comment by foolip** [comment]
I believe this will fix https://github.com/web-platform-tests/wpt-pr-bot/issues/69. But it might be a good idea to merge https://github.com/web-platform-tests/wpt-pr-bot/pull/118 and add some minimal test coverage for index.js first.

**Comment by foolip** [comment]
@zcorpan per discussion with @boazsender I'm reassigning this and two other of my PRs to you. Not super high priority but would be nice to get it done within a few weeks.

**Review by zcorpan — APPROVED** [review]

**Comment by zcorpan** [comment]
In

```

**Comment by foolip** [comment]
@zcorpan sweet, thanks! Does everything still seem to be working for a new and updated PR?

**Comment by zcorpan** [comment]
Yes, I think so. I haven't tested updating PRs, but the papertrail shows recent activity for e.g. https://github.com/web-platform-tests/wpt/pull/20441
